azure-pipelines.yml

```diff
@@ -29,3 +29,8 @@ steps:
   inputs:
     command: 'custom'
     customCommand: 'run build'
+- task: Npm@1
+  displayName: Test
+  inputs:
+    command: 'custom'
+    customCommand: 'run test'
```

Comment:
There's a way for the test results to show in ADO. I'm not sure what needs to be done but we probably want these captured as "test results"